react/src/components/search-app/SearchBarComponent.jsx

```diff
@@ -28,12 +28,12 @@ export default class SearchBarComponent extends React.Component {
     this.setState({ query });
     if (query.length >= 2) {
       const http = new XMLHttpRequest();
-      const params = JSON.stringify({ query });
-      http.open('POST', '/api/v1/search', true);
+      http.open('GET', `${this.props.host}/search?query=${encodeURIComponent(query)}&mode=auto-suggest`, true);
```

Comment:
Makes sense. You could also achieve that by specifying default props. Something like:
```
SearchBarComponent.defaultProps = {
  host: constants.host
};
```
And 3rd party users can still override it with their own value. That being said, I'm fine with your solution, because I can't think of any advantage/disadvantage. I was just curious, thanks for elaborating.